app/models/tag.rb

```diff
@@ -17,7 +18,9 @@ def self.find(repository:, name:)
   def initialize(**args)
     super
 
-    self.manifests = fetch_manifests.sort_by(&:architecture)
+    self.manifests = fetch_manifests.find_all { |manifest| 
+      manifest.architecture() != "unknown" && manifest.os() != "unknown"
```

Comment:
Just one nit:

```suggestion
      manifest.architecture != "unknown" && manifest.os != "unknown"
```

Background: `()` are optional for method calls in ruby when there are no arguments. It's really just a cosmetic thing but most ruby devs (including me) prefer it that way :)

Reply:
Changed as requested.